packages/fetch-plus-plus/package.json

```
@@ -0,0 +1,31 @@
+{
+  "name": "fetch-plus-plus",
+  "version": "1.0.0",
+  "description": "A simple isomorphic wrapper around fetch with retries and error handling",
+  "keywords": [
+    "fetch",
+    "node",
+    "isomorphic",
+    "retry",
+    "error"
+  ],
+  "bugs": "https://github.com/bloq/js-packages/issues",
+  "license": "MIT",
+  "author": {
+    "name": "Gonzalo D'Elia",
+    "email": "gonzalo@bloq.com"
+  },
+  "main": "src/index.js",
+  "dependencies": {
+    "http-errors": "2.0.0",
+    "isomorphic-fetch": "3.0.0",
+    "p-retry": "4.6.1"
```

Packages should not have pinned versions. Let's allow minor and patched dependencies to be used too.

```suggestion
    "http-errors": "^2.0.0",
    "isomorphic-fetch": "^3.0.0",
    "p-retry": "^4.6.1"
```

---

Versions unpinned here [2a572e3](https://github.com/bloq/js-packages/pull/5/commits/2a572e33bba0998b024a697fd0a52e0e6bb6efe0)